lib/MainPageHelpers/thanksListWidget.dart

```diff
@@ -103,6 +106,10 @@ class _ThanksListWidgetState extends State<ThanksListWidget> {
         1) {
       showThankYouPopup(userInfoProvider);
     }
+    AnalyticsService mixPanelService = GetIt.instance<AnalyticsService>();
+    mixPanelService.trackEvent(
+      "Item added to Gratitude Journal",
```

Comment:
Consider using more programmatically friendly event name. that will allow easier copy-pasting the name for various filters